Cu 86b0dyx4n create login form carmen peterson

varsvisualizes opened the pull request:
Started building Login page with React

varsvisualizes committed:
Update App.js

Added 'Login' to the Login route so it didn't have the same route as the Home page

varsvisualizes committed:
Added Login.jsx and Login.css

Starting to build out Login page with React

allantito commented:
Task linked: [CU-86b0dyx4n Create Login Form](https://app.clickup.com/t/86b0dyx4n)

varsvisualizes committed:
Delete P1/Frontend/entertainment-app/src/pages/Login.jsx

Duplicate file compared to Login.js

varsvisualizes committed:
Update Login.js

Updated with the info I had previously in Login.jsx

varsvisualizes committed:
Update App.js

'Login' on the route is now 'login' lowercase to match the rest of the routes

varsvisualizes committed:
Update Login.css

varsvisualizes committed:
Update Login.css

varsvisualizes committed:
Update Login.css

Centered box within the screen; added flexbox direction so the SVG will stack directly on top of the Login box

varsvisualizes committed:
Update Login.js

Added 'center-box' div for centering the items in the screen

varsvisualizes committed:
Updated Login and CSS, moved Outfit
